proto-lens-protoc/src/Data/ProtoLens/Compiler/Combinators.hs

```diff
@@ -359,3 +359,7 @@ modifyModuleName :: (String -> String) -> ModuleName -> ModuleName
 modifyModuleName f (Syntax.ModuleName _ unpacked) =
   Syntax.ModuleName () $ f unpacked
 
+
+modifyIdent :: (String -> String) -> Name -> Name
```

Comment:
Code structure nit: we try to define names inside the Definitions module and use them in Generate.  In this case, adding new fields to `EnumInfo` with the `Name`s for the unrecognized types:
https://github.com/google/proto-lens/blob/master/proto-lens-protoc/src/Data/ProtoLens/Compiler/Definitions.hs#L157